tests/components/subaru/test_sensor.py

```diff
@@ -143,7 +143,7 @@ def _assert_data(hass, expected_state):
     expected_states = {}
     for item in sensor_list:
         expected_states[
-            f"sensor.{slugify(f'{TEST_DEVICE_NAME} {item.name}')}"
```

Comment:
I also must say, I don't like this change how I did this, since this is only working since every translation is the same as it's key ("rear left" -> "rear_left"), if your key would now differ from the key ("rl" -> "rear left" -> "rear_left") it would break I think